c2corg_api/views/route.py

```diff
@@ -63,7 +63,7 @@ def validate_main_waypoint(is_on_create, request, **kwargs):
 
     # no association found
     request.errors.add(
-        'body', 'main_waypoint_id', 'no association for the main waypoint')
```

Comment:
must check the translation on the frontend part when merged

Reply:
I didn't found any translation for this error

Reply:
thus it is not translated by the frontend yet 🙂